### PR TITLE
[german] de/basics/imports

### PR DESCRIPTION
**Pull request by wilzbach**

### CODE REVIEW

**Comment by wilzbach** [comment]
German reviewer roulette: @WebFreak001 could you act as a second pair of eyes on this translation?

File: public/content/de/basics/imports-and-modules.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Imports und Module
+
+Für ein einfaches "Hallo Welt" Program in D werden Imports benötigt.
```
Comment (WebFreak001) [inline public/content/de/basics/imports-and-modules.md]: `Programm`, nicht `Program`
http://www.duden.de/rechtschreibung/Programm
https://de.wikipedia.org/wiki/Computerprogramm

**Comment by WebFreak001** [comment]
Looks good otherwise, just that small english/german mix up

**Comment by wilzbach** [comment]
> Looks good otherwise, just that small english/german mix up

Wow that was quick. Thanks a lot :)
The current policy here is to merge once a reviewer looked at the changes and gives his final okay. I know it's trivial in this case, but I still think it's a good protocol. Would you be so kind to give your final okay?

File: public/content/de/basics/imports-and-modules.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Imports und Module
+
+Für ein einfaches "Hallo Welt" Programm in D werden Imports benötigt.
+Mit `import` werden alle öffentlichen Funktionen und Typen des __Modules__ verfügbar.
+
+Die Standardbibliothek [Phobos](https://dlang.org/phobos/) ist unter
+dem **Packet** `std` vorhanden und diese Module können per `import std.MODULE`
```
Comment (WebFreak001) [inline public/content/de/basics/imports-and-modules.md]: `... und dessen Module ...` fits better in this case I think

Comment (wilzbach) [inline public/content/de/basics/imports-and-modules.md]: or maybe even "ihre Module"? But that sounds  strange :/

Comment (WebFreak001) [inline public/content/de/basics/imports-and-modules.md]: Try `Die Standardbibliothek Phobos ist unter dem Packet std vorhanden. Die einzelnen Module können per import std.MODULE importiert werden`

**Comment by wilzbach** [comment]
> Try Die Standardbibliothek Phobos ist unter dem Packet std vorhanden. Die einzelnen Module können per import std.MODULE importiert werden

Thanks. Sounds nice to me :)

**Comment by WebFreak001** [comment]
ok looks all good to me

**Pull request merged**